### PR TITLE
Clarify SignalR scaleout

### PR DESCRIPTION
**Pull request by boblangley**
An alternative to #4292

### CODE REVIEW

**Comment by boblangley** [comment]
My apologies, I didn't see #4292 before I submitted this. Take what you want.

File: samples/near-realtime-clients/sample.md [hunk 1]
```diff
@@ -65,6 +65,68 @@ In this sample, the SignalR client is implemented as a .NET console application.
...
+SS1-->|SignalR Message|CB
+SS2-->|SignalR Message|CC
+SS2-->|SignalR Message|CD
+```
+
+NOTE: In this case each NServiceBus subscriber will require a unique endpoint name to receive a copy of the published message and push the event to its connected clients.
+
+### Using a SignalR Backplane
+
+If SignalR usage extends beyond simply pushing NServiceBus events, then scaling out using a backplane will likely be necessary. In this case, only deploy a _single_ NServiceBus subscriber forwarding events to SignalR. This will prevent duplicates, at the expense of an additional message hop as the message is forwarded through the backplane to the clients, since each SignalR server will forward pushed events to other SignalR servers on the backplane.
```
Comment (SzymonPobiega) [inline samples/near-realtime-clients/sample.md]: Would it makes sense to deploy multiple instances of that subscriber for HA reasons? Should we mention it here?

File: samples/near-realtime-clients/sample.md [hunk 2]
```diff
@@ -65,6 +65,68 @@ In this sample, the SignalR client is implemented as a .NET console application.
 
 When the number of connected clients exceeds the capability of a single SignalR server, it will be necessary to scale out the SignalR server. Scaling out SignalR is described in the [Introduction to Scaleout in SignalR](https://docs.microsoft.com/en-us/aspnet/signalr/overview/performance/scaleout-in-signalr) article by Microsoft.
 
+### Simple NServiceBus Event Forwarding (No SignalR Backplane)
```
[the remark this answers is not in the document]
Comment (boblangley) [reply samples/near-realtime-clients/sample.md]: It is the logical next step if you are only using SignalR to propagate NServiceBus events. We can remove it.

**Comment by boblangley** [comment]
I removed the simple scenario, and made some updates to the diagram for the backplane scenario with a description.

**Pull request merged**